WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -779,7 +779,7 @@ public void onGoogleLoginFinished() {
     @Override
     public void onGoogleSignupFinished(String name, String email, String photoUrl, String username) {
         AnalyticsTracker.track(AnalyticsTracker.Stat.SIGNUP_SOCIAL_SUCCESS);
-
+        mLoginAnalyticsListener.trackAnalyticsSignIn(false);
```

Comment:
Shouldn't this be `trackAnalyticsSignIn(true)`? I believe the flag is meant to identify self-hosted vs WP.com logins, and WP.com signups should count as the latter.

Reply:
(The same would apply to the change in `WPMainActivity`.)

Reply:
you're right of course! good catch